images/Dockerfile.opensuse-tumbleweed

```diff
@@ -1,8 +1,7 @@
 ARG BASE_IMAGE=opensuse/tumbleweed
 
-ARG K3S_VERSION
-
 FROM $BASE_IMAGE
+ARG K3S_VERSION
```

Comment:
we can drop this, it is a leftover - not really used

Reply:
I know I know.. not part of these changes =) .. buuut

Reply:
yeah I was wondering why this were being used.....will drop them